src/importexport/mei/CMakeLists.txt

```diff
@@ -0,0 +1,48 @@
+# SPDX-License-Identifier: GPL-3.0-only
+# MuseScore-CLA-applies
+#
+# MuseScore
+# Music Composition & Notation
+#
+# Copyright (C) 2021 MuseScore BVBA and others
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 3 as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+set(MODULE iex_mei)
+
+include_directories(${CMAKE_CURRENT_LIST_DIR}/internal/)
+include_directories(${CMAKE_CURRENT_LIST_DIR}/thirdparty/)
+include_directories(${CMAKE_CURRENT_LIST_DIR}/thirdparty/libmei/)
```

Comment:
The problem is for includes from within the library, since we do not have explicit paths there and it would be preferable not to change them. I changed to setting `MODULE_INCLUDE`